CONTRIBUTING.md

```diff
@@ -290,7 +289,9 @@ This can be achieved with GitHub's [_squash and merge_](https://help.github.com/
 
 ### Commit Message
 
-To make the single commit expressive, its message must be detailed and [good]((http://chris.beams.io/posts/git-commit/)) (really, read that post!).
+Because of the noise it generates on the issue, please do _not_ mention the issue number in the message.
```

Comment:
Move this here from the "Commit" section; however this kind of contradicts with the commit message format described below.

Reply:
I would suggest to move it to the end of this section and rephrase it as follows:

```text
Finally, because of the noise it generates on the issue, please do _not_ mention the issue number in the message during development.
```